t/00-use.t

```diff
@@ -0,0 +1,21 @@
+#===============================================================================
+#
+#         FILE: 00-use.t
+#
+#  DESCRIPTION: Use the module, check it compiles, check module version
+#
+#       AUTHOR: Pete Houston (pete), cpan@openstrike.co.uk
+#
+#===============================================================================
+
+use strict;
+use warnings;
+
+use Test::More tests => 3;
+
+BEGIN {
```

Comment:
No reason to make this a `BEGIN`.

Reply:
And actually, this test serves no serious purpose. The VERSION would need updating every time. Please delete it.

Reply:
I've tidied up t/00-use.t but perhaps you meant the entire script should be deleted instead?

Reply:
Yes, I am saying the whole file should be deleted (sorry!)

Reply:
No worries! It's been removed now in 63455ae